nyxx_interactions/lib/interactions.dart

```diff
@@ -1,40 +1,39 @@
 library nyxx_interactions;
```

Comment:
have you check if whole thing is working properly? By changing commands, only perms and perms with commands?
Most of the api isn't public so I suppose we cannot test that?

Reply:
I'm going to test once #172 has been fixed. As that may be an issue during testing